src/System.Private.CoreLib/src/System/Reflection/MemberInfoSerializationHolder.cs

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// 
+
+using System;
+using System.Runtime.Remoting;
+using System.Runtime.Serialization;
+using System.Globalization;
+using System.Diagnostics.Contracts;
+
+namespace System.Reflection 
+{   
+    [Serializable]
+    internal class MemberInfoSerializationHolder : ISerializable, IObjectReference 
+    {
+        #region Staitc Public Members
+        public static void GetSerializationInfo(SerializationInfo info, String name, RuntimeType reflectedClass, String signature, MemberTypes type)
+        {
+            GetSerializationInfo(info, name, reflectedClass, signature, null, type, null);
+        }
+
+        public static void GetSerializationInfo(
+            SerializationInfo info,
+            String name,
+            RuntimeType reflectedClass,
+            String signature,
+            String signature2,
+            MemberTypes type,
+            Type[] genericArguments)
+        {
+            if (info == null)
+                throw new ArgumentNullException("info");
```

Comment:
@AtsushiKan I think you got a stale copy of CoreCLR repo. This is `nameof(info)` in the current master.